.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI build and push
+
+concurrency:
+  group: ci-${{ github.run_id }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - main
+      - v[0-9]+
+      - v[0-9]+.[0-9]+
+      - cryostat-v[0-9]+.[0-9]+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      CRYOSTAT_GRAFANA_IMG: quay.io/cryostat/cryostat-grafana-dashboard
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        repository: ${{ github.event.pull_request.head.repo.full_name }}
```

Comment:
Since its a push, we probably don't have to specify this? Also, `github.event.pull_request` will be undefined here?